articles.tex

```diff
@@ -680,4 +680,36 @@
 The Judicial Board may present an official ruling following the Judicial Investigation.
 This ruling may be appealed in the same manner as an Executive Board decision \ref{Appeals}.
 
+% ARTICLE VII - TEMPORARY AMENDMENTS
+\article{Temporary Amendments}
+In the case that an amendment should be temporary, the amendment's changes should be listed here. 
+Each of these amendments is given a designated period of time, and will cease to exist immediately.
+If at any point interpretation of the changes needs to be made, the Executive Board will use the process defined in \ref{Judicial} to come to an agreement on how to proceed. 
+Occasionaly, Temporary Amendments may become permanent through a referendum, which is described in detail as part of each Temporary Amendment. 
+\asection{Changes to Evaluations}
+During the 2018-2019 period the following will supercede the text otherwise found in this document.
+\asubsection{Changes}
+\asubsubsection{Removal of Introductory Project}
+\ref{The Introductory Project} will no longer be a requirement as described in \ref{The Introductory Process}. 
+Additionally, \ref{Introductory Executive Board} will also no longer be elected or perform any duties during the described period. 
+\asubsubsection{Change of Packet Percentage}
+\ref{Creation of Accounts} currently stipulates that Introductory Members must acquire 60\% (rounded up to the nearest whole person) of required signatures (excluding those of Resident members who have not passed a Membership Evaluation) in the Introductory Packet or successfully complete Introductory Evaluations. 
+During the described period, the percentage of required signatures in the Introductory Packet is increased to 75\%. 
+\asubsubsection{Changes to Packet Times}
+Introductory Members will recieve an \ref{The Introductory Packet} during the second week of each semester. 
```

Comment:
At what point during week 2 will the packet be handed out? This language allows packets to be handed out both at the start and the end of the week.

Reply:
This may be the same thing that @devinmatte mentioned in his comment, but I'd change this to "the second week after their application is accepted", or something to that effect - this fails to take into account anybody whose application process starts after the beginning of the semester.

Reply:
This wording is intended to allow different scenarios to be handled in similar ways. 

Applicants accepted during the evals meeting of the second week would be given a packet and have until the end of the third week to finish it. Applicants before the 2nd week would begin during the beginning of the week. 

Any applicants after week 2 will not receive a packet until the following semester.

Reply:
If we are trying to drive recruitment over the course of the week, this doesn't really work. An applicant would have to wait for an evals meeting to start packet. So if they show up to an event after an evals meeting and want to join, they can't get a packet. 
I would really like a more specific wording for this section.